ons_alpha/core/blocks/stream_blocks.py

```diff
@@ -13,8 +14,22 @@
     PanelBlock,
     RelatedContentBlock,
     RelatedLinksBlock,
-    TableBlock,
 )
+from ons_alpha.core.blocks import (
+    TableBlock as OldTableBlock,
+)
+
+
+class ONSTableBlock(StructBlock):
+    heading = CharBlock(required=True, help_text="Add a heading for the table.")
+    table = TableBlock(required=True, help_text="Add the table data here.")
+    source = CharBlock(required=False, help_text="Add the source of the table data if applicable.")
+    footnotes = RichTextBlock(features=["bold", "italic"], required=False, help_text="Add any footnotes for the table.")
```

Comment:
```suggestion
    footnotes = RichTextBlock(features=settings.RICH_TEXT_BASIC, required=False, help_text="Add any footnotes for the table.")
```

with the corresponding import of `from django.conf import settings`